docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -21,64 +21,56 @@ tags:
 
 :::tip Support, stability, and dependency info
 
-- In [Public Preview](/evaluate/development-production-features/release-stages#public-preview)
-
-:::
+- API Keys used for Cloud Operations Tools (tcld, API, Terraform) are in [Public Preview](/evaluate/development-production-features/release-stages#public-preview)
+- API Keys used for Namespace Authentication are in [Pre Release](/evaluate/development-production-features/release-stages#pre-release)
+  :::
 
 Temporal Cloud supports secure programmatic access through API Key Authentication.
 
 ## Overview
 
-Temporal Cloud API Keys facilitate user-level authentication.
-Each key is unique to a single user and ties directly to their Role-Based Access Control (RBAC) settings, ensuring secure and appropriate access.
+Temporal Cloud API Keys provide identity based authentication where an Identity can be a Temporal User or [Service Account](/cloud/service-accounts)
```

Comment:
```suggestion
Temporal Cloud API Keys provide identity-based authentication where an Identity can be either a Temporal User or a [Service Account](/cloud/service-accounts).
```

Reply:
Identity is not listed as one of our uppercase terms. If so, please file a ticket so I can update our linter checks (probably as a warning) or consider lower-case.

Comma before where